pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -454,11 +454,12 @@ def _update_learning_rates(
                 self.scheduler_progress.increment_ready()
 
                 # update LR
-                if lr_scheduler["reduce_on_plateau"]:
-                    lr_scheduler["scheduler"].step(monitor_val)
-                else:
-                    lr_scheduler["scheduler"].step()
-
+                self.trainer._call_lightning_module_hook(
+                    "lr_scheduler_step",
+                    lr_scheduler["scheduler"],
+                    lr_scheduler["opt_idx"] or 0,
```

Comment:
will it be `None` in any case? @carmocca 
https://github.com/PyTorchLightning/pytorch-lightning/blob/b56d8677ad0ff8513e566334f4a78a24b88480c3/pytorch_lightning/core/optimizer.py#L345-L358

Reply:
Nope, I added it thinking mypy would complain

But it doesn't at the moment, so we can remove it.

```suggestion
                    lr_scheduler["opt_idx"],
```

It should since `lr_scheduler.opt_idx` is an `Optional[int]` because it doesn't know the `opt_idx` was set.